packages/babel-core/test/api.js

```diff
@@ -1,16 +1,15 @@
 import * as babel from "../lib/index";
 import sourceMap from "source-map";
-import assert from "assert";
 import path from "path";
 import Plugin from "../lib/config/plugin";
 import generator from "@babel/generator";
 
 function assertIgnored(result) {
-  assert.ok(!result);
+  expect(!result).toBeTruthy();
```

Comment:
this probably should at least get reversed to
```js
expect(result).toBeFalsy();
```

Reply:
I agree. :-) Will make the change in sometime.